tools/tpm2_loadexternal.c

```diff
@@ -338,9 +369,7 @@ static tool_rc check_options(ESYS_CONTEXT *ectx) {
      * in leui of AES or vice versa.
      */
     if (!ctx.key_type && ctx.private_key_path) {
-        LOG_ERR("Expected key type via -G option when specifying private"
-                " portion of object");
-        return tool_rc_option_error;
+        ctx.is_tsspem = true;
```

Comment:
shouldn't the determination of is_tsspem come from the tpm2_util_object_load and set in the  tpm2_loaded_object structure?

Reply:
I was under the impression that not specifying the key type(-G) and specifying the private portion only would mean that it is a TSS private pem file.

The tpm2_loaded_object doesn't appear in the tpm_loadexternal_ctx structure, so I had to add the bool directly in the tpm_loadexternal_ctx structure.

Reply:
This is a useful error message that you can move to label tss2_err if you are unable to load the file as tsspriv pem.

Reply:
Here specify ctx.is_tsspem = (!ctx.key_type && ctx.private_key_pat);